nifi-nar-bundles/nifi-smb-bundle/pom.xml

```diff
@@ -26,7 +26,36 @@
     <packaging>pom</packaging>
 
     <modules>
+        <module>nifi-smb-connection-pool-api</module>
+        <module>nifi-smb-connection-pool-api-nar</module>
+        <module>nifi-smb-connection-pool</module>
+        <module>nifi-smb-connection-pool-nar</module>
         <module>nifi-smb-processors</module>
         <module>nifi-smb-nar</module>
     </modules>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.hierynomus</groupId>
+                <artifactId>smbj</artifactId>
+                <version>0.11.5</version>
+            </dependency>
+            <dependency>
+                <groupId>net.engio</groupId>
+                <artifactId>mbassador</artifactId>
+                <version>1.3.0</version>
```

Comment:
The latest version appears to be 1.3.2, is there a reason for not using that version?